app/views/laptops/new.html.slim

```diff
@@ -0,0 +1,7 @@
+.container
+  h1 New laptop
+
+  .form
+    = render 'form'
+
+   	= link_to 'Back', laptops_path, class: 'btn btn-primary'
```

Comment:
If the link_to back is supposed to be underneath and separate to the form, then the = link_to must have the same indentation level as the above = render 'form' line.
It mustn't be indented more.